packages/ui/src/views/unified-pipeline-studio/components/palette-drawer/components/step-palette-content-layout.tsx

```diff
@@ -18,7 +20,7 @@ const StepsPaletteContentLayout = {
   },
 
   SectionItem: function SectionHeader({ children }: { children: React.ReactNode }) {
-    return <div className="hover:bg-shade-10 mb-3 flex flex-col rounded-md border p-2">{children}</div>
+    return <div className=" mb-3 flex flex-col rounded-md border p-2 hover:!bg-background-4">{children}</div>
```

Comment:
not: remove unintended leading space `mb-3`

Reply:
Not entirely sure if `hover:!bg-background-4` works as intended here. Tailwind itself doesn’t have `!important` support natively in hover states unless configured explicitly.

Reply:
fixed, it does work as intended for the moment